examples/fastrp-and-knn.ipynb

```diff
@@ -139,16 +210,30 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Run FastRP and mutate projection with result\n",
+    "# Now let's run FastRP and mutate our projected graph 'purchases' with the results\n",
     "result = gds.fastRP.mutate(\n",
     "    G,\n",
     "    mutateProperty=\"embedding\",\n",
+    "    randomSeed=42,\n",
     "    embeddingDimension=4,\n",
     "    relationshipWeightProperty=\"amount\",\n",
     "    iterationWeights=[0.8, 1, 1, 1],\n",
     ")\n",
     "\n",
-    "print(f\"Embedding vectors produced: {result['nodePropertiesWritten']}\")"
+    "# Let's make sure we got an embedding for each node\n",
+    "print(f\"Number of embedding vectors produced: {result['nodePropertiesWritten']}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1213e261",
+   "metadata": {},
+   "source": [
+    "## Similarities with kNN\n",
+    "\n",
+    "Now we can run [kNN](https://neo4j.com/docs/graph-data-science/current/algorithms/knn/) to identify similar nodes by using the node embeddings that we generated with FastRP as `nodeProperties`. Since we are working with a small graph, we can set `sampleRate` to 1 and `deltaThreshold` to 0 without having to worry about long computation times. The `concurrency` parameter is set to 1 (along with the fixed `randomSeed`) in order to get a deterministic result. Please see [the syntax section of the kNN documentation](https://neo4j.com/docs/graph-data-science/current/algorithms/knn/#algorithms-knn-syntax) for more information on these parameters.\n",
```

Comment:
Some of the explanations below relies on rather specific outcomes, so I think we better keep it. And since we explain it here, I think that's fine.

This text is copied from the GDS manual and I'drather keep it the same for now